packages/block-editor/src/components/use-setting/index.js

```diff
@@ -115,13 +115,26 @@ export default function useSetting( path ) {
 			}
 			const settings = select( blockEditorStore ).getSettings();
 
+			// 0 - Use settings for this block instance, if there's any.
+			//     Also, look up in the block hierarchy.
```

Comment:
Makes sense, but the docstring for `useSetting` could use an update. It was lacking as it were, but now we should make it clear that it is a block-contextual hook.